Orange/data/tests/test_pandas.py

```diff
@@ -240,9 +229,21 @@ def test_table_from_frame_time(self):
         self.assertEqual(table.domain.variables[0].have_time, 1)
         self.assertEqual(table.domain.variables[0].have_date, 0)
 
-    def test_table_from_frame_datetime(self):
-        from Orange.data.pandas_compat import table_from_frame
+    def test_version(self):
+        """
+        When this test starts to fail, a new pandas version has been released.
+        - check if the issue has been fixed:
+            - pd.DataFrame([[pd.Timestamp("01:02:03.5")]])
+            - # this should output:
+                                    0
+            0 2023-04-04 01:02:03.500
+
+        - if it has, remove line 202 in this file and uncomment lines 660
+        and 661 in test_variable
+        """
+        self.assertTrue(pd.__version__ <= "2.0.0")
 
```

Comment:
```suggestion
```

Is this one necessary since you mentioned it works with Pandas master version and we skip 2.0.0 in requirements?